plugin/src/main/java/com/denizenscript/denizen/objects/properties/PropertyRegistry.java

```diff
@@ -144,6 +144,7 @@ public static void registermainProperties() {
         PropertyParser.registerProperty(ItemUnbreakable.class, ItemTag.class);
 
         // register core MaterialTag properties
+        PropertyParser.registerProperty(MaterialMode.class, MaterialTag.class);

```

Comment:
This is on very much the wrong line

Reply:
Which line does it go on?

Reply:
Look at the other properties and think about it